models/turbine_models/custom_models/sd_inference/utils.py

```diff
@@ -10,38 +10,19 @@
 )
 
 # If flags are verified to work on a specific model and improve performance without regressing numerics, add them to this dictionary. If you are working with bleeding edge flags, please add them manually with the --ireec_flags argument.
-gfx94X_flags = {
-    "all": [
+amdgpu_flags = {
+    "all": [],
+    "unet": [
         "--iree-global-opt-propagate-transposes=true",
         "--iree-opt-outer-dim-concat=true",
-        "--iree-rocm-bc-dir=/opt/rocm/amdgcn/bitcode",
         "--iree-vm-target-truncate-unsupported-floats",
         "--iree-llvmgpu-enable-prefetch=true",
-        "--verify=false",
-        "--iree-rocm-waves-per-eu=2",
         "--iree-opt-data-tiling=false",
-        "--iree-codegen-log-swizzle-tile=4",
-        "--iree-llvmgpu-promote-filter=true",
-        "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline, iree-preprocessing-pad-to-intrinsics)",
-    ],
-    "unet": [
-        "--iree-codegen-llvmgpu-use-conv-vector-distribute-pipeline",
-        "--iree-codegen-llvmgpu-reduce-skinny-matmuls",
         "--iree-codegen-gpu-native-math-precision=true",
-        "--iree-codegen-llvmgpu-use-vector-distribution",
-        "--iree-codegen-winograd-use-forall",
-    ],
-    "clip": [
-        "--iree-codegen-llvmgpu-use-vector-distribution",
-        "--iree-codegen-llvmgpu-reduce-skinny-matmuls",
-        "--iree-global-opt-only-sink-transposes=true",
-    ],
-    "vae": [
-        "--iree-codegen-llvmgpu-use-conv-vector-distribute-pipeline",
-        "--iree-codegen-llvmgpu-use-vector-distribution",
-        "--iree-global-opt-only-sink-transposes=true",
-        "--iree-codegen-winograd-use-forall",
+        "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline, iree-preprocessing-pad-to-intrinsics)",
     ],
+    "clip": [],
+    "vae": ["--verify=false"],
```

Comment:
We really shouldn't be relying on this. Do we still need this flag?

Reply:
I have a comment about this over at https://github.com/nod-ai/SHARK-TestSuite/pull/194#discussion_r1576670475 too:

> `--verify=false` is suuuper sketchy in tests. We generally recommend it for _already tested_ programs as a way to improve compilation time. Let's see how slow the CI is with verification on and then decide what we want to do.

If this is meant for testing then I'd prefer to keep verification on (to catch issues and not let unverified IR slip through the pipeline). If this is for users/developers and the compilation is already tested/trusted, then turning off verification can speed up compilation time rather significantly.

Reply:
> I have a comment about this over at [nod-ai/SHARK-TestSuite#194 (comment)](https://github.com/nod-ai/SHARK-TestSuite/pull/194#discussion_r1576670475) too:
> 
> > `--verify=false` is suuuper sketchy in tests. We generally recommend it for _already tested_ programs as a way to improve compilation time. Let's see how slow the CI is with verification on and then decide what we want to do.
> 
> If this is meant for testing then I'd prefer to keep verification on (to catch issues and not let unverified IR slip through the pipeline). If this is for users/developers and the compilation is already tested/trusted, then turning off verification can speed up compilation time rather significantly.

Makes sense. We can set it to false on the user-facing side and keep it on for testing/dev.